client/src/types/index.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/no-unresolved
+import { GeoJSON } from 'geojson';
```

Comment:
Shouldn't make a difference here (since this is just `@types/geojson`), but FYI, if you're just importing types, you should use the following syntax, so that any unnecessary files don't get bundled.
```suggestion
import type { GeoJSON } from 'geojson';
```